### DISCUSSION TITLE
Exporting Animation to .fbx

### DISCUSSION THREAD

**Discussion by Dallacoosta** [neverhood311/Stop-motion-OBJ#172] [Q&A]
Hello!

I'm a beginner. so I'm not sure if this functionality doesn't exist or I'm just doing something wrong. I import my .plys and everything works fine. However, after baking and exporting, the animation doesn't move anymore. it is always frozen in the same frame. Could somebody please give me an idea of how to properly do it? would be very kind! 

best Regards 
Diego

**Comment by neverhood311** [comment]
As mentioned in the [wiki](https://github.com/neverhood311/Stop-motion-OBJ/wiki#exporting-to-alembic-andor-fbx-doesnt-work), export sequences isn't officially supported. And not for lack of effort or interest. Users have been requesting FBX export since [2017](https://github.com/neverhood311/Stop-motion-OBJ/issues/8) and it's been requested [several](https://github.com/neverhood311/Stop-motion-OBJ/issues/22) [times](https://github.com/neverhood311/Stop-motion-OBJ/issues/46) [since](https://github.com/neverhood311/Stop-motion-OBJ/issues/163).

All of the links I've shared contain a little bit of the full story.

To make a long story short, there's a problem with Blender's FBX exporter that prevents me from properly exporting multiple meshes that hide and appear on keyframes.

If you can use Alembic, some users have had luck with that. The details are in one of the links I shared.
